lib/macho/load_commands.rb

```diff
@@ -204,15 +97,30 @@ def to_s
     # pretend that strings stored in LCs are immediately available without
     # explicit operations on the raw Mach-O data.
     class LCStr
+      # @return [Fixnum] the number of null bytes used to pad the string
+      attr_reader :npad
```

Comment:
Similarly to `cmdsize`, I don't think the string needs to know its own padding. That's assuming too much (among other things that there's only exactly one `LCStr` in any given load command).